Lombiq.Tests.UI.Samples/Tests/BasicOrchardFeaturesTests.cs

```diff
@@ -25,7 +25,17 @@ public BasicOrchardFeaturesTests(ITestOutputHelper testOutputHelper)
     public Task BasicOrchardFeaturesShouldWork(Browser browser) =>
         ExecuteTestAsync(
             context => context.TestBasicOrchardFeaturesAsync(RecipeIds.BasicOrchardFeaturesTests),
-            browser);
+            browser,
+            configuration =>
+            {
+                // The UI Testing Toolbox includes DbCommand execution counter, after the end of test it checks the
+                // number of executed commands with the same Sql command and parameter set against the threshold value
```

Comment:
It's only really the SQL command, regardless of the parameters, that we need to count. A fully repeated query is an issue, but it's a subset of duplicated queries, especially if we want to guard against `SELECT` N+1 queries (where the same query is executed repeatedly, but with different parameters).

Reply:
I'm not sure if this is addressed. Aren't still only duplications with the same query and parameters counted?

Reply:
You can configure both. `CounterThresholdConfiguration.DbCommandExecutionThreshold` is used against the count of the same command text with the same parameter value(s), while `CounterThresholdConfiguration.DbCommandTextExecutionThreshold` is used against the count of the same command text.

Reply:
I see, OK then.